src/css/color-picker.less

```diff
@@ -150,3 +150,10 @@
   position: relative;
   z-index: 1;
 }
+
+// Eyeballed to be the same height as color preview and width when there are 4
+// colors per row.
+.color-picker-hex-input.form-control.input-sm {
+  width: 104px;
+  height: 28px;
+}
```

Comment:
yea we need to override the values for form-control.input-sm, but there are other styles that we inherit from them.

We could re-write it to add all the styles we need, but thought this was a ittle bit easier.